media/plg_media-action_rotate/js/rotate.es6.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+Joomla = window.Joomla || {};
+
+Joomla.MediaManager = Joomla.MediaManager || {};
+Joomla.MediaManager.Edit = Joomla.MediaManager.Edit || {};
+
+(function mediaActionRotate() {
```

Comment:
here we can also use arrow function 
```
(() => {
...
})()
```

Reply:
I do not understand when to use arrow functions and when to use named functions. Wouldn't it be better to leave the name of the function? If we leave off the function name then when the function throws an exception we do not see a name in the stack trace.